libcontainer/cgroups/fs2/stats_util_test.go

```diff
@@ -0,0 +1,77 @@
+package fs2
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/opencontainers/runc/libcontainer/cgroups"
+)
+
+func expectMemoryStatEquals(t *testing.T, expected, actual cgroups.MemoryStats) {
```

Comment:
Same here: maybe move this to fscommon_test and make available for both fs and fs2?

If you'll do something like this, please do it as a separate commit.